src/components/core/Reloader.react.js

```diff
@@ -51,7 +51,7 @@ class Reloader extends React.Component {
                 ) {
                     // Look if it was a css file.
                     let was_css = false;
-                    for (let a of reloadRequest.content.files) {
+                    for (const a of reloadRequest.content.files) {
```

Comment:
I think that is an error, make sure it works because I kinda remember putting const and it failed when there was multiple files. There can be multiple files if multiple assets files are saved at the same time.

Reply:
This must be some artifact from a merge with master, I haven't touched any of the reloading code as far as I remember.

Reply:
I haven't had a chance to rebase yet, because I'm still working on fixing the tests

Reply:
I think it's prettier that changes it, I remember it changed it also when I ran it, maybe add an exclude here.

Reply:
Still need to change this back.